main/package.mill

```diff
@@ -90,6 +90,7 @@ object `package` extends RootModule with build.MillStableScalaModule with BuildI
 
     def ivyDeps = Agg(
       build.Deps.osLib,
+      build.Deps.sbtIo,
```

Comment:
What's this `sbtIo` dependency for?

Reply:
I no longer need it, was using it to expand glob patterns, will remove